public/content/documentation/native/patterns/tidbit.md

```diff
@@ -0,0 +1,271 @@
+
+   - WHEN I swipe to navigate
+        - THEN the focus should move to the tidbit or tidbit CTA
+            - AND the name and role should be expressed
+   - WHEN I double-tap the tidbit CTA
+        - THEN the CTA should be activated
+
+3. Scenario: Test screen reader output on all devices 
+
+   - WHEN a screen reader is active
+        - THEN the purpose should be clear and match the visible label for the CTA
+            - AND all text and meaningful images should be announced
+            - AND the CTA should be identified as a button in iOS
+            - AND the CTA should be identified as a button or "double tap to activate" in Android
+            - AND the CTA’s state (DISABLED/DIMMED) should be expressed
+
+4. Scenario: Test device OS settings for text resize 
+
+   - WHEN I have increased text size up to 200% in the device settings
+        - THEN text should resize up to 200% without losing information 
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/radio-button](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/radio-button)
+
+## Videos
+
+### iOS VoiceOver
+
+<video controls>
+  <source src="media/video/native/tidbit/tidbit_IosVoiceOver.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Android Talkback
+<video controls>
+  <source src="media/video/native/tidbit/tidbit_AndroidTalkback.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+## iOS Developer Notes
+
+   - Initial screen reader focus can be on the entire tidbit and announce everything with or without a Call To Action (CTA). Or, the text and CTA can have separate focus.
+   - Meaningful images should have an accessibility label
+  
+### Name
+
+   - Programmatic name describes the purpose of the control (if CTA is available)
+   - If visible text label exists, the programmatic name should match the visible text label.
+        - **Note:** Setting a programmatic name while a visible text label exists may cause VoiceOver to duplicate the announcement of the name. If this happens, hide the visible text label from VoiceOver recognition.
+
+#### UIKit
+
+   - You can programmatically set the visible label with `setTitle()`.
+        - The _component title will overwrite the _component `accessibilityLabel`.
+   - If a visible label is not applicable in this case, set the _component `accessibilityLabel` to the label of your choice.
+        - To do this in Interface Builder, set the label using the Identity Inspector
+   - To hide labels from VoiceOver programmatically, set the label's `isAccessibilityElement` property to `false`
+   - To hide labels from VoiceOver using Interface Builder, uncheck `Accessibility Enabled` in the Identity Inspector.
+
+#### SwiftUI
+
+   - If no visible label, use view modifier `accessibilityLabel(_:)`
+
+### Role
+
+   - Please use guidance for the Button control if a CTA is applicable
```

Comment:
could be helpful to link over to this component. Leave as is if you prefer not to.